src/test/java/reposense/authorship/FileInfoExtractorTest.java

```diff
@@ -25,6 +25,7 @@
     private static final String WINDOWS_ILLEGAL_FILE_NAME_BRANCH = "windows-illegal-filename";
     private static final String EDITED_FILE_INFO_BRANCH = "getEditedFileInfos-test";
     private static final String DIRECTORY_WITH_VALID_WHITELISTED_NAME_BRANCH = "directory-with-valid-whitelisted-name";
+    private static final String BRANCH_WITH_VALID_WHITELISTED_FILE_NAME_BRANCH = "535-FileInfoExtractorTest-branchWithValidWhitelistedFileName.txt";
```

Comment:
This is causing checkstyle to fail in Travis and Appveyor as the line is greater than 120 characters.
Make a line break after the `=`